### PR TITLE
docs: add configuration migration to the migration guide

### PR DESCRIPTION
**Pull request by swiatekm-sumo**
Add a brief description on how to use the configuration migration tool.

I also moved the full list of changes to the end of the document, as it's not that useful for someone starting the migration. I'd like to restructure it a bit more to make it clear for users what they need to care about based on their configuration.

Instead, the start has a short `Important Changes` section that provides context for the rest of the upgrade steps.

### CODE REVIEW

File: docs/v3-migration-doc.md [hunk 1]
```diff
@@ -12,112 +11,62 @@
...
+## Important changes
 
-- Upgrading kube-prometheus stack
+### OpenTelemetry Collector
 
-  We are updating Kube-prometheus-stack to newest available version.
-  Major feature related to that change is upgrading kube-state-metrics to v2
+The new version replaces both Fluentd and Fluent Bit with the OpenTelemetry Collector. In the majority of cases, this doesn't
+require any manual intervention. However, custom processing in Fluentd or Fluent Bit will need to be ported to the OpenTelemetry Collector
+configuration format. See below for details.
```
Comment (sumo-drosiek) [inline docs/v3-migration-doc.md:27]: ca nwe link it to the section? `See below for details.`

File: docs/v3-migration-doc.md [hunk 2]
```diff
@@ -12,112 +11,62 @@
...
-### Requirements
+- At this point you should have `new-values.yaml` in your working directory which
+  can be used for the upgrade:
 
-- `helm3`
-- `kubectl`
-- `jq`
+  ```bash
+  helm upgrade <RELEASE-NAME> sumologic/sumologic --version=3.0.0 -f new_values.yaml
+  ```
```
Comment (sumo-drosiek) [inline docs/v3-migration-doc.md:69]: I would add note to look at the migration script output if any manual intervation is needed

File: docs/v3-migration-doc.md (hunk 1 above)
Comment (swiatekm-sumo) [reply docs/v3-migration-doc.md:27]: I will once that section actually exists. I'm going to add it in a new PR.

File: docs/v3-migration-doc.md (hunk 2 above)
Comment (swiatekm-sumo) [reply docs/v3-migration-doc.md:69]: Honestly, it's probably better not to have this here, as users will run it before they carry out the necessary manual steps from the next section. So this should probably be moved to the end.

Comment (swiatekm-sumo) [reply docs/v3-migration-doc.md]: I moved it to the end and mentioned the output

**Review by sumo-drosiek — APPROVED** [review]
LGTM, but would be nice to have second approval from @SumoLogic/open-source-collection-team

**Review by kkujawa-sumo — APPROVED** [review]

**Pull request merged**